### PR TITLE
HPCC-31249 Add HINTS for clienttools location for build-assets builds

### PR DESCRIPTION
**Pull request by Michael-Gardner**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -190,14 +190,16 @@ SET(CT_ARCH "i386")
 IF (EMBED_64BIT_CLIENTTOOLS)
   SET(CT_ARCH "x86_64")
 ENDIF ()
-find_file(CLIENTTOOLS_PACKAGE_FILE "hpccsystems-clienttools-community_${version}-${stagever}${CMAKE_SYSTEM_NAME}-${CT_ARCH}.exe" HINTS ${CMAKE_CURRENT_BINARY_DIR}/../HPCC-Platform )
+find_file(CLIENTTOOLS_PACKAGE_FILE "hpccsystems-clienttools-community_${version}-${stagever}${CMAKE_SYSTEM_NAME}-${CT_ARCH}.exe" HINTS ${CMAKE_CURRENT_BINARY_DIR}/../HPCC-Platform ${CMAKE_CURRENT_BINARY_DIR}/../.. ${CMAKE_CURRENT_BINARY_DIR}/.. )
```
Comment (Michael-Gardner) [inline CMakeLists.txt:193]: Added hints that will allow us to find the clienttools package in github actions build-asset workflow

**Comment by Michael-Gardner** [comment]
Test build here:  Output for the eclide package should be +100MB https://github.com/Michael-Gardner/HPCC-Platform/actions/runs/7874284284

File: CMakeLists.txt [hunk 2]
```diff
@@ -190,14 +190,16 @@ SET(CT_ARCH "i386")
...
 MESSAGE ("-- Clienttools package: ${CLIENTTOOLS_PACKAGE_FILE}")
 if (CLIENTTOOLS_PACKAGE_FILE)
     install ( PROGRAMS ${CLIENTTOOLS_PACKAGE_FILE} DESTINATION tmp )
     get_filename_component(CLIENTTOOLS_PACKAGE_FILE_NAME ${CLIENTTOOLS_PACKAGE_FILE} NAME)
     list ( APPEND CPACK_NSIS_EXTRA_INSTALL_COMMANDS "
         ExecWait '$INSTDIR\\\\tmp\\\\${CLIENTTOOLS_PACKAGE_FILE_NAME} /S'
 ")
+else (CLIENTTOOLS_PACKAGE_FILE)
+    MESSAGE(FATAL_ERROR "Could not locate hpccsystems-clienttools-community_${version}-${stagever}${CMAKE_SYSTEM_NAME}-${CT_ARCH}.exe")
```
[the remark this answers is not in the document]
Comment (GordonSmith) [reply CMakeLists.txt:202]: This will be very inconvenient for the eclide developers...

**Review by GordonSmith — CHANGES_REQUESTED** [review]

**Comment by Michael-Gardner** [comment]
@GordonSmith Removed that FATAL_ERROR message

**Review by GordonSmith — APPROVED** [review]

**Pull request merged**